### DISCUSSION TITLE
[Q&A] Which fluentd package is compatible for HPUX servers

### DISCUSSION THREAD

**Discussion by Evalanidis** [fluent/fluentd#4556] [Q&A]
### What is a problem?

Hello all,

We have a HPUX server and we want to install fluentd so we can send logs to kibana.
Based the information I see here :https://www.fluentd.org/download/fluent_package i dont see HPUX as any of the compatible platforms. 

Is there any package that might be compatible with HPUX server? 

Thank you in advance.

### Environment

```markdown
- Fluentd version:
- TD Agent version:
- Fluent Package version:
- Docker image (tag):
- Operating system:
- Kernel version:
```

**Comment by kenhys** [comment]
There is no package for HPUX server so you might want to setup with gem.

https://docs.fluentd.org/installation/install-by-gem
